File: 6.C# Fundamentals/OOP Basics/2.Defining_Classes_Exce/DefiningClassesExce/DefiningClassesExce/Person.cs
public class Person
{
    public Person()
    {}

    public Person(string name, int age)
    {
        this.age = age;
        this.name = name;
    }

    private string name;
    private int age;

    public string Name
    {
        get { return this.name; }
        set
        {
            this.name = value;
        }
    }

    public int Age
    {
        get { return this.age; }
        set
        {
            this.age = value;
        }
    }

}

File: 8.JS Web/ExpressJS Fundamentals/02.Intro to Note.js/Person.js

class Person {
    
    constructor(name, age, gender){
        this.name = name;
        this.age = age;
        this.gender = gender;
    }

    SayHello(){
        console.log(`Hello, my name is ${this.name}, my gender is ${this.gender} and I am ${this.age} years old.`);
    }
}


module.exports = Person;